metadata-jobs/mae-consumer-job/src/main/java/com/linkedin/metadata/kafka/MaeConsumerApplication.java

```diff
@@ -9,11 +9,11 @@
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.FilterType;
 
-
 @SuppressWarnings("checkstyle:HideUtilityClassConstructor")
 @SpringBootApplication(exclude = {ElasticsearchRestClientAutoConfiguration.class, CassandraAutoConfiguration.class,
     SolrHealthContributorAutoConfiguration.class})
-@ComponentScan(excludeFilters = {
+@ComponentScan(basePackages = {"com.linkedin.gms.factory.config"},
```

Comment:
Nice! We will need to watch out for existing long-running PRs that make changes to the consumers if they create classes outside of this base package and expect the beans to be loaded.